core/include/kernel_defines.h

```diff
@@ -187,6 +187,20 @@ extern "C" {
  */
 #define IS_USED(module) IS_ACTIVE(module)
 
+/**
+ * @def         KERNEL_VERSION(mayor, minor, patch, extra)
+ * @brief       Generates a 64 bit variable of a release version
+ *              Comparisons to this only apply to released branches
+ *
+ * @param[in]   a   Mayor version of the release
+ * @param[in]   b   Minor version of the release
+ * @param[in]   c   Patch level of the release
+ * @param[in]   d   Extraversion, user defined
```

Comment:
The names in the `@def` should also be reflected in the `@params` (and the macro itself) + the spell checker does not know what an Extraversion is (which to my knowledge either is not an English word ;-P). Also, you want "major" as in "greater in rank" not the "elected chief executive of a town or city" ;-).

```suggestion
 * @def         KERNEL_VERSION(major, minor, patch, extra)
 * @brief       Generates a 64 bit variable of a release version
 *              Comparisons to this only apply to released branches
 *
 * @param[in]   major   Mayor version of the release
 * @param[in]   minor   Minor version of the release
 * @param[in]   patch   Patch level of the release
 * @param[in]   extra   Extra version, user defined
```

Reply:
> Also, you want "major" as in "greater in rank" not the "elected chief executive of a town or city"

heh I must have gotten confused because Berlin is about to bump it's mayor version ;-)